app/test_main.py

```diff
@@ -1,3 +1,25 @@
 from app.main import get_human_age
 
-# write your code here
+
```

Comment:
Consider removing any unnecessary comments or prints before committing your code, as per the 'Clean Code' checklist item.